src/template.rs

```diff
@@ -27,14 +27,18 @@ fn engine() -> liquid::Parser {
 }
 
 pub fn create_liquid_object(
+    args: &GenerateArgs,
     project_dir: &Path,
     name: &ProjectName,
     crate_type: &CrateType,
-    force: bool,
 ) -> Result<Object> {
     let authors: Authors = get_authors()?;
     let os_arch = get_os_arch();
-    let project_name = if force { name.raw() } else { name.kebab_case() };
+    let project_name = if args.force {
```

Comment:
Nit: do you know the `.then()` method on bool? It's a beautiful way to handle this thing..

Reply:
fixed.

I knew it, but didn't think to change the line because:
* it was already there
* I don't think `then()` on bool is especially readable

anyhow - its done!